repository-data/application/src/main/resources/hcm-config/namespaces/hee/guidanceDocument.yaml

```diff
@@ -0,0 +1,165 @@
+definitions:
+  config:
+    /hippo:namespaces/hee/guidance:
+      jcr:primaryType: hipposysedit:templatetype
+      jcr:mixinTypes: ['editor:editable', 'mix:referenceable']
+      jcr:uuid: 75f31343-4f4b-4aa6-8e80-28067c96ae65
+      /hipposysedit:nodetype:
+        jcr:primaryType: hippo:handle
+        jcr:mixinTypes: ['mix:referenceable']
+        jcr:uuid: 4ee4631b-efde-4f48-bcfa-f85feb78d3f3
+        /hipposysedit:nodetype:
+          jcr:primaryType: hipposysedit:nodetype
+          jcr:mixinTypes: ['mix:referenceable', 'hipposysedit:remodel']
+          jcr:uuid: a038628e-1009-4e15-8e80-20903a157921
+          hipposysedit:node: true
+          hipposysedit:supertype: ['hee:basedocument', 'hippostd:relaxed', 'hippotranslation:translated']
+          hipposysedit:uri: http://www.heecmsplatform.com/hee/nt/1.0
+          /title:
+            jcr:primaryType: hipposysedit:field
+            hipposysedit:mandatory: false
+            hipposysedit:multiple: false
+            hipposysedit:ordered: false
+            hipposysedit:path: hee:title
+            hipposysedit:primary: false
+            hipposysedit:type: String
+            hipposysedit:validators: [required]
+          /summary:
+            jcr:primaryType: hipposysedit:field
+            hipposysedit:mandatory: false
+            hipposysedit:multiple: false
+            hipposysedit:ordered: false
+            hipposysedit:path: hee:summary
+            hipposysedit:primary: false
+            hipposysedit:type: Text
+            hipposysedit:validators: [required]
+          /body:
+            jcr:primaryType: hipposysedit:field
+            hipposysedit:mandatory: false
+            hipposysedit:multiple: false
+            hipposysedit:ordered: false
+            hipposysedit:path: hee:body
+            hipposysedit:primary: false
+            hipposysedit:type: hippostd:html
+          /quickLinks:
+            jcr:primaryType: hipposysedit:field
+            hipposysedit:mandatory: false
+            hipposysedit:multiple: true
+            hipposysedit:ordered: true
+            hipposysedit:path: hee:quickLinks
+            hipposysedit:primary: false
+            hipposysedit:type: hippo:mirror
+          /calendardate:
+            jcr:primaryType: hipposysedit:field
+            hipposysedit:mandatory: false
+            hipposysedit:multiple: false
+            hipposysedit:ordered: false
+            hipposysedit:path: hee:lastReviewed
+            hipposysedit:primary: false
+            hipposysedit:type: CalendarDate
+          /calendardate_:
+            jcr:primaryType: hipposysedit:field
+            hipposysedit:mandatory: false
+            hipposysedit:multiple: false
+            hipposysedit:ordered: false
+            hipposysedit:path: hee:nextReviewed
+            hipposysedit:primary: false
+            hipposysedit:type: CalendarDate
```

Comment:
@smanivasag should we combine nextReviewed and lastReviewed into a compound type because they will be always come together ? So easier to be reused on other page type.

Reply:
Sure, go ahead. Thanks!